module.json

```diff
@@ -83,6 +83,27 @@
                         },
                         "label" : "__switches_label__",
                         "order" : 2
+                     },
+                     "notifications": {
+                        "fields": {
+                           "item": {
+                              "fields": {
+                                 "device":{
+                                    "type": "select",
+                                    "label": "__notifications_helper__",
+                                    "datasource": "namespaces",
+                                    "field": "optionLabels",
+                                    "optionLabels": "namespaces:devices_toggleButton:notification:deviceName"
+                                 },
+                                 "message":{
+                                    "label":"__notifications_message__",
+                                    "type": "textarea"
```

Comment:
Maybe add a "helper" field that indicates that the message will only be used if the event does not have its own message